tests/tape_tests/test_ensemble.py

```diff
@@ -67,6 +67,49 @@ def test_from_parquet(data_fixture, request):
         # Check to make sure the critical quantity labels are bound to real columns
         assert parquet_ensemble._source[col] is not None
 
+ 
+@pytest.mark.parametrize(
+    "data_fixture",
+    [
+        "parquet_files_and_ensemble_without_client",
+    ],
```

Comment:
Is the plan to add more data fixtures in the future? With only one data fixture, you don't need parametrize.

Or is this needed in order to read out the multiple parameters from parquet_files_and_ensemble_without_client?

Reply:
The current plan is to to add more data fixtures, though we can clean this up now if preferred.

Reply:
Fine to leave it as-is. I just wanted to check that it had a future purpose.